src/php/composer.json

```diff
@@ -1,20 +1,24 @@
 {
-  "name": "grpc/grpc-dev",
-  "description": "gRPC library for PHP - for Developement use only",
+  "name": "grpc/grpc",
+  "type": "library",
+  "description": "gRPC library for PHP",
+  "keywords": ["rpc"],
+  "homepage": "https://grpc.io",
   "license": "Apache-2.0",
   "version": "1.7.0",
   "require": {
     "php": ">=5.5.0",
-    "google/protobuf": "^v3.3.0"
+    "google/protobuf": "^3.4.0"
   },
   "require-dev": {
     "google/auth": "v0.9"
   },
+  "suggest": {
+    "ext-protobuf": "For better performance, install the protobuf C extension."
+  },
   "autoload": {
     "psr-4": {
-      "Grpc\\": "lib/Grpc/",
-      "": ["tests/interop/",
```

Comment:
We also need these 2 lines back. There are several tests that rely on these. We can list out the namespaces in a separate enhancement

Reply:
This should be moved to `autoload-dev`